samples/WebAPI_Marten/GreetingsEntities/Person.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+
+namespace GreetingsEntities
+{
+    public class Person
+    {
+        public int Id { get; set; }
+        public ICollection<Greeting> Greetings { get; set; }
```

Comment:
In case I would like to use private setters here so I can follow some DDD/OOP practices; do I have to use reflection somehow so Marten can work with serialization?

Reply:
Thanks, this one helped https://martendb.io/v3/documentation/documents/json/newtonsoft